client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -29,14 +32,13 @@ const AddGoogleAppsCard = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'email' ) ],
 
 	render() {
-		const gapps = this.props.products.gapps,
+		const prices = get( this.props, 'products.gapps.prices', [] ),
```

Comment:
I'm not sure if this is an issue or not, but it appears that prices for GAPPS are only available in for AUD, BRL, CAD, EUR, GBP, JPY, and USD. So, if I have my currency set to SEK, HUF, ILS, MXN, or NZD, then `monthlyPrice` and `annualPrice` will end up `null`.